lib/replicate.js

```diff
@@ -201,6 +202,15 @@ Peer.prototype.onrequest = function (request) {
   }
 }
 
+Peer.prototype._updateOptions = function () {
+  if (this.ack || (this.feed.extensions.length)) {
```

Comment:
What does an options update have to do with the `ack` flag?

Reply:
It needs to send an options message if ack is enabled OR you are using extensions, othersise it is not needed

Reply:
Seems like ack could be implemented as an extension, although might be more work than it's worth

Reply:
Yea, we should consider that down the line I think